src/Tasks/CreateVisualBasicManifestResourceName.cs

```diff
@@ -146,6 +146,10 @@ TaskLoggingHelper log
                     manifestName.Append(rootNamespace).Append(".");
                 }
 
+                // Replace spaces in the directory name with underscores. Needed for compatibility with Everett.
+                // Note that spaces in the file name itself are preserved.
+                string everettCompatibleDirectoryName = MakeValidEverettIdentifier(Path.GetDirectoryName(info.cultureNeutralFilename));
```

Comment:
Can you put this next to where you actually use it? Also, if this is appended, will it not make a poorly-formed path? If rootNamespace is `C:\`, and info.cultureNeutralFileName is at `C:\Users\forgind\Desktop\folder on desktop\`, then I think manifestName (right after the Replace) would be `C:.C:.Users.forgind.Desktop.folder_on_desktop`. Am I misunderstanding something?